index.html

```diff
@@ -1951,8 +1950,7 @@ <h3>
           The <dfn>orientation</dfn> member is a <a>string</a> that serves as
           the <a>default orientation</a> for all <a>top-level browsing
           contexts</a> of the web application. The possible values are those of
-          the <a>OrientationLockType</a> enum defined in
-          [[SCREEN-ORIENTATION]].
+          the {{orientationlocktype}} enum defined in [[SCREEN-ORIENTATION]].
```

Comment:
```suggestion
          the {{OrientationLockType}} enum defined in [[SCREEN-ORIENTATION]].
```

Reply:
What you have here is correct, but it's wrong in the xref database :) I'll get that fixed as it should be `OrientationLockType`.